dev.rollup.config.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-env node */
+'use strict';
+
+import pkg from './package.json';
+import os from 'os';
+import fs from 'fs-extra';
+import path from 'path';
+import glob from 'glob';
+import serve from 'rollup-plugin-serve';
+import livereload from 'rollup-plugin-livereload';
+import { umdConfig } from './rollup.config';
+
+const BROWSER_FILENAME = `${pkg.name}.js`;
+const tmpDestPath = fs.mkdtempSync(path.join(os.tmpdir(), `${pkg.name}-dev-server-`));
+
+let httpsConfig;
+try {
+    httpsConfig = {
+        key: fs.readFileSync('sslcert-dev/key.pem'),
+        cert: fs.readFileSync('sslcert-dev/cert.pem')
+    };
+} catch (e) {
+    console.warn('Unable to find ssl key/cert for https, serving over http.');
+}
+
+const transformExampleSdkUrl = (buffer) => {
+    return buffer.replace(
+        /(\s*<script.*src=")([^"]+client-app-sdk[^"]+)(".*<\/script>\s*)/i,
+        '$1' + (BROWSER_FILENAME || '$2') + '$3\n'
+    );
+};
+
+const buildExample = (relativeFilePath) => {
+    fs.outputFileSync(
+        path.join(tmpDestPath, relativeFilePath.replace('examples/', '')),
+        transformExampleSdkUrl(fs.readFileSync(relativeFilePath, 'utf8'))
+    );
+};
+
+// Build all examples initially
+glob.sync('examples/**/*', { nodir: true }).forEach(buildExample);
+
+export default Object.assign({}, umdConfig, {
+    output: Object.assign({}, umdConfig.output, {
+        dir: tmpDestPath
+    }),
+    watch: {
+        clearScreen: false
```

Comment:
One thing I just added with the latest commit (to warn and fall back to HTTP) was to add this option here. Rollup was clearing the terminal on every build and all of our other logs were getting tossed. I'm open to leaving this as it was, we just wouldn't be able to have any other logs in our dev server.

Reply:
Seems fine to let the logs run to me.